src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/CacheClearAction.java

```diff
@@ -120,4 +118,39 @@ public CacheClearAction build() {
       return new CacheClearAction(appContext, coolOffPeriodInMillis, canUpdate);
     }
   }
+
+  public static class Summary {
```

Comment:
Can we make this as an util class or rename as CacheClearActionSummary?

Reply:
Since this Summary is only used as a Json container within each action class(similar to the Builder class within the action class), I would like to leave it as it is. We can make them as separate classes if needed in the future